protocols/api_health_checker.py

```
@@ -2,46 +2,51 @@
+                    "name": endpoint["name"],
+                    "status": response.status_code,
+                    "response_time_ms": round(response_time, 2),
+                    "healthy": response.status_code == 200,
+                }
+            )
+
             if response.status_code != 200:
                 failures += 1
-                
+
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)